src/cljs/rems/administration/create_resource.cljs

```diff
@@ -7,52 +7,30 @@
             [rems.autocomplete :as autocomplete]
             [rems.collapsible :as collapsible]
             [rems.spinner :as spinner]
-            [rems.status-modal :refer [status-modal]]
+            [rems.status-modal :as status-modal]
             [rems.text :refer [text localize-item]]
-            [rems.util :refer [dispatch! fetch post!]]))
+            [rems.util :refer [dispatch! fetch post!]]
+            [rems.status-modal :as status-modal]))
 
-(defn- reset-form [db]
-  (assoc db
-         ::form {:licenses #{}}
-         ::loading? true))
 
 (rf/reg-event-fx
  ::enter-page
  (fn [{:keys [db]}]
-   {:db (reset-form db)
+   {:db (assoc db
+         ::form {:licenses #{}}
+         ::loading? true)
     ::fetch-licenses nil}))
 
-(rf/reg-sub
- ::loading?
- (fn [db _]
-   (::loading? db)))
+(rf/reg-sub ::loading? (fn [db _] (::loading? db)))
 
 ;; form state
 
-(rf/reg-sub
```

Comment:
meta: all reformattings like this should've been a separate (subsequent) PR to reduce line noise